src/tokens/utilities.ts

```diff
@@ -1,5 +1,17 @@
+import type {TokenGroup} from './tokens';
+
 const BASE_FONT_SIZE = 16;
 
-export function rem(px: string) {
-  return `${parseInt(px, 10) / BASE_FONT_SIZE}rem`;
+function rem(value: string) {
+  return value.replace(
+    // https://regex101.com/r/RBL7EE/1
+    /\d+(?:\.\d+|\d*)px/g,
+    (px: string) => `${parseInt(px, 10) / BASE_FONT_SIZE}rem`,
```

Comment:
I believe we need to update this to `parseFloat`. I will circle back to this before merging the PR.

Reply:
`parseFloat` does add more precision to the `rem` function, but we currently have no need for it at this time. We may want to consider re-enabling this when we revisit responsive and/or container based queries to additionally include floating point values for print media.

Here is the resulting before and after with `border-radius-1` set to `4.1px`
![image](https://user-images.githubusercontent.com/32409546/145633425-08578436-b499-4887-8cf9-707531eeb274.png)